### ISSUE TITLE
Add modulo (%) infix operator

### ISSUE THREAD

**Issue by fananek** [vapor/leaf-kit#87]
# Description
Modulo (%) operation is not supported at the moment. It can be used e.g. for calculation even/odd rows. Zebra striped tables or grids are pretty common design requirement.

**example psudo-code:**
```html
#for(row in rows)
  #if(index % 2 == 0):
    // use row backgroud color
  #else:
    // use row alternate backgroud color
  #endif
#endfor
```

# Solution
Implement function which handles modulo operation for `Int` as well as `Double` input values.
*Already working on pull-request.*

**Comment by grundoon** [comment]
(@fananek just a pointer to how I implemented it for Leaf 1.x https://github.com/vapor/template-kit/pull/19)

**Comment by fananek** [comment]
@grundoon Thanks, but it's not relevant anymore. Leaf has been completely rebuilt. I have it almost ready for Leaf 4. Just making few more tests.

**Comment by 0xTim** [comment]
Closed via #88
